runtime/arc.js

```diff
@@ -102,11 +102,12 @@ class Arc {
     return [...this._viewsById.values()];
   }
 
-  clone() {
+  // Makes a copy of the arc used for speculative execution.
+  makeSpeculative() {
```

Comment:
i was debating about the name
- makeSpeculativeClone: seems too long
- cloneSpeculative: reads as if we were cloning an already speculative arc
and in general, we're only cloning some of the fields, so maybe "clone" is not the right word, WDYT?